supervisor/shared/filesystem.c

```diff
@@ -71,10 +71,7 @@ static void make_empty_file(FATFS *fatfs, const char *path) {
 
 
 static void make_sample_code_file(FATFS *fatfs) {
-    #if CIRCUITPY_FULL_BUILD == 0
-    make_empty_file(fatfs, "/code.py");
-
-    #else
+    #if CIRCUITPY_FULL_BUILD == 1
```

Comment:
The canonical way to do this is just `#if CIRCUITPY_FULL_BUILD`, i.e. just non-zero, not explicitly testing for `1`

Reply:
@dhalbert sure, I thought other non-zero values could yield different built types.